src/components/Editorial/Content.tsx

```diff
@@ -0,0 +1,37 @@
+import { useTheme } from '@mui/material';
+import Stack from '@mui/material/Stack';
+import Typography from '@mui/material/Typography';
+import { type CommonProps } from 'types/components';
+
+export interface EditorialContentProps extends CommonProps {
+  title: string;
+  eyelet?: string;
+  body: string | JSX.Element;
+}
+
+export const Content = ({
+  eyelet,
+  title,
+  body,
+  theme,
+}: EditorialContentProps) => {
+  const { palette } = useTheme();
+  const eyeletColor =
+    theme === 'dark' ? palette.primary.contrastText : palette.text.secondary;
+  const textColor =
+    theme === 'dark' ? palette.primary.contrastText : palette.text.primary;
+
+  return (
+    <Stack maxWidth={{ md: '448px' }} gap="16px">
```

Comment:
Isn't better to set a width using MUI function prop? (rif https://mui.com/material-ui/customization/breakpoints/#api)

Reply:
Using theme theme `breakpoints.values.md / 2` instead of `448px`